build-libssp.sh

```diff
@@ -37,36 +37,18 @@ export PATH="$PREFIX/bin:$PATH"
 : ${ARCHS:=${TOOLCHAIN_ARCHS-i686 x86_64 armv7 aarch64}}
 
 if [ ! -d libssp ]; then
-    svn checkout -q https://github.com/gcc-mirror/gcc/tags/releases/gcc-7.3.0/libssp
+    svn checkout -q svn://gcc.gnu.org/svn/gcc/tags/gcc_9_2_0_release/libssp 
```

Comment:
Can you make a separate commit for switching back from the github mirror to gcc.gnu.org first, before this change? Essentially that would be a revert of ad3d710c9a0fe40193b01c4a87fb0cde7fdf9718, with a commit message saying that it probably only was a temporary outage in the svn service back when I made that change.

Reply:
OK